resources/testdriver.js

```diff
@@ -1066,6 +1066,27 @@
          */
         clear_device_posture: function(context=null) {
             return window.test_driver_internal.clear_device_posture(context);
+        },
+
+        /**
+         * Run the deletion algorithm for all hosts in the stateful bounce
```

Comment:
What does deletion algorithm refer to here? https://privacycg.github.io/nav-tracking-mitigations/ only uses that phrase once, in a note in https://privacycg.github.io/nav-tracking-mitigations/#bounce-tracking-mitigations-timers.

The WebDriver command calls https://privacycg.github.io/nav-tracking-mitigations/#bounce-tracking-timer so I guess that's the algorithm.

Could this be documented in terms more like what the spec uses, like "Run bounce tracking mitigations"?